### DISCUSSION TITLE
Where can I get the RAINBOW_PROVIDER_API_KEY

### DISCUSSION THREAD

**Discussion by sj719045032** [rainbow-me/rainbowkit#2003] [Feedback]
I want to contribute some code, but when I install the dependency I get an error without RAINBOW_PROVIDER_API_KEY. How do I get it?
<img width="942" alt="image" src="https://github.com/rainbow-me/rainbowkit/assets/7941762/61f5269c-b933-4446-a32d-dffe910d745a">

**Comment by DanielSinclair** [comment]
@sj719045032 You can just export an empty variable to ignore this while building locally:
```bash
export RAINBOW_PROVIDER_API_KEY=""
```

**Reply by PPnaruto** [reply]
In which file i need to specify this ? I tried to set up this by adding .env in the packages/rainbowkit , but still getting same error

**Reply by harked** [reply]
yes, me too

**Reply by DanielSinclair** [reply]
This is no longer required. Give it another try!

**Comment by omartabibi** [comment]
Is RAINBOW_PROVIDER_API_KEY considered secret and should not be publicly available like in js files?

**Reply by DanielSinclair** [reply]
Nope, no impact or security issues for dApps.
